doc/modules/preprocessing.rst

```diff
@@ -423,3 +423,24 @@ values than observed values.
 
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`example_missing_values.py`
+
+
+Custom Transformers
+===================
+
+Often, you will want to convert an existing python function into a transformer
+to assist in data cleaning or processing. Users may implement a transformer from
```

Comment:
I don't agree that it's most valuable with a search over the function. But I think it has no value except in a `Pipeline` or `FeatureUnion` and it should be illustrated in that context.